### DISCUSSION TITLE
User group and welcome email sending

### DISCUSSION THREAD

**Discussion by cairoapcampos** [requarks/wiki#5494] [Help / Questions] [answered]
When I create a new user, I can't add them directly to the "Administrators" group, nor can I send the welcome email. Options are disabled.

Accessing groups it is possible to add the user.

Version: Wiki.js 2.5.284 in Docker.

**Comment by NGPixel** [comment, marked as answer]
This is normal. Because Administrators is a sensitive group, you can only add users to it from the group page itself.

The Send welcome email option is disabled because it's not implemented yet.

**Reply by skollnope** [reply]
Do you know when this feature could be available ? thank you :)

**Comment by cairoapcampos** [comment]
Thanks for the answer. I'm using Wiki.js at the company I work for knowledge management. Great tool.
